docs/docs/dev_docs/sandbox/main.md

```diff
@@ -2,51 +2,376 @@
+  private-token Retrieving unencrypted logs for block 3 +4ms
+  private-token Emitted logs:  [ 'Balance set in constructor' ] +5ms
+  private-token Alice's balance 1000000 +4s
+  private-token Bob's balance 0 +3s
+  private-token Transferring 543 tokens from Alice to Bob... +0ms
+  private-token Retrieving unencrypted logs for block 4 +20s
+  private-token Emitted logs:  [ 'Coins transferred' ] +13ms
+  private-token Alice's balance 999457 +4s
+  private-token Bob's balance 543 +3s
+```
+
+Here, we used the same contract abstraction as was previously used for reading Alice's balance. But this time we called `send()` generating and sending a transaction to the network. After waiting for the transaction to settle we were able to retrieve the newly emitted unencrypted logs and check the new balance values.
+
+Finally, the contract has a `mint` function that can be used to generate new tokens for an account. This takes 2 arguments:
+
+1. The quantity of tokens to be minted.
+2. The recipient of the new tokens.
+
+<GithubCode owner="AztecProtocol" language="rust" repo="aztec-packages" branch="master" filePath="yarn-project/noir-contracts/src/contracts/private_token_contract/src/main.nr" startLine={48} endLine={66} />
+
+Let's mint some tokens to Bob's account:
+
+#include_code Mint /docs/src/code_examples/sandbox_example.ts typescript
+
+Our complete output should now be:
+
+```
+  private-token Aztec Sandbox Info  { version: 1, chainId: 31337 } +0ms
+  private-token Creating accounts using schnorr signers... +2ms
+  private-token Created Alice's account at 0x054d89d0...f17e +23s
+  private-token Created Bob's account at 0x0a8410a1...7c48 +1ms
+  private-token Deploying private token contract minting an initial 1000000 tokens to Alice... +0ms
+  private-token Transaction status is mined +8s
+  private-token Contract successfully deployed at address 0x143e0af4...11b6 +7ms
+  private-token Retrieving unencrypted logs for block 3 +4ms
+  private-token Emitted logs:  [ 'Balance set in constructor' ] +5ms
+  private-token Alice's balance 1000000 +4s
+  private-token Bob's balance 0 +3s
+  private-token Transferring 543 tokens from Alice to Bob... +0ms
+  private-token Retrieving unencrypted logs for block 4 +20s
+  private-token Emitted logs:  [ 'Coins transferred' ] +13ms
+  private-token Alice's balance 999457 +4s
+  private-token Bob's balance 543 +3s
+  private-token Minting 10000 tokens to Bob... +1ms
+  private-token Retrieving unencrypted logs for block 5 +13s
+  private-token Emitted logs:  [ 'Coins minted' ] +13ms
+  private-token Alice's balance 999457 +4s
+  private-token Bob's balance 10543 +4s
+```
+
+That's it! We have successfully deployed a private token contract to an instance of the Aztec network and mined private state-transitioning transactions. We have also queried the resulting state all via the interfaces provided by the contract.
+
+## Accounts and Keys
```

Comment:
I kind of felt the same about that section. The reason I used 2 accounts is that without it you can't see the results of the transfer, i.e. Bob's balance.

It's a compromise really. In a private transaction you can only see one side of it but that doesn't make a very good demo!

Reply:
> but that doesn't make a very good demo

I disagree - I think it's a great demo for a private-first chain! Otherwise this demo is exactly the same as a regular public ERC20.